pages/docs/manual/latest/migrate-to-v11.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: "Migrate to v11"
+description: "Instructions on upgrading to ReScript 11 with the new uncurried mode"
+canonical: "/docs/manual/latest/migrate-to-v11"
+---
+
+# Migrate to ReScript 11
+
+## Foreword
+
+The ReScript community is proud to introduce ReScript V11 which comes with a ton of new features but also removes a lot of bulk.
+A migration to it can be very straightforward, but it can also take some time, depending on your code style or what dependencies you use.
+
+Please have a look at the full [set of breaking changes](#list-of-all-breaking-changes) below to be able to decide whether this is a task you want to undertake. There is also the possibilty to opt-out of uncurried mode for now, which is probably the most fundamental change of this release. That and other new and notable features are discussed in the following blogposts:
+
+- [Better interop with customizable variants](/blog/improving-interop)
+- [Enhanced Ergonomics for Record Types](/blog/enhanced-ergonomics-for-record-types)
+- [First-class Dynamic Import Support](/blog/first-class-dynamic-import-support)
+- [Uncurried Mode](/blog/uncurried-mode)
+
+## Minimal Migration
```

Comment:
I am not sure this should be at the most prominent place.
Can we move it to the bottom and link to it from the sentence with "opt-out of uncurried mode" above?